demo.py

```diff
@@ -1,7 +1,7 @@
 #1
 # Sets a default curve (secp256k1)
 import random
-from umbral import umbral, keys, config
+from umbral import pre, keys, config
```

Comment:
This namespace is more accurate, IMO. :+1:

Reply:
:heavy_check_mark: